pysb/export/json.py

```diff
@@ -243,7 +243,8 @@ def encode_model(cls, model):
         d = super(PySBJSONWithNetworkEncoder, cls).encode_model(model)
 
         # Ensure network generation has taken place
-        generate_equations(model)
+        if not model.odes:
```

Comment:
`generate_equations` already internally checks if the rxn network has been generated, so these `if` statements aren't required

Reply:
that's right